source/_integrations/vicare.markdown

```diff
@@ -24,51 +25,86 @@ ha_codeowners:
+{% endnote %}
+
+{% include integrations/config_flow.md %}
+
+## Entities
+
+ViCare represents devices as a set of [data points](https://documentation.viessmann.com/static/iot/data-points) and the ViCare {% term integration %} maps those to {% term entity entities %} of different {% term platform platforms %} in Home Assistant. A single device may be represented by one or more platforms.
+
+### Climate
+
+Represents the heating controls of your device.
+
+{% note %}
+Viessmann devices with room temperature sensing will show the current room temperature via `current_temperature`. All other devices will show the current supply temperature of the heating circuit.
+{% endnote %}
+
+### Water heater
+
+Represents the domestic hot water controls of your device.
+
+It is not possible to turn on/off water heating via the water heater {% term integration %} since this would conflict with the operation modes of the heating integration. Therefore, the operation mode of that integration is just available as an attribute and cannot be modified.
+
+### Sensor
+
+Additional data for a device is available as separate sensors. The sensors are automatically discovered based on the available API data points.
+
+### Button
+
+Button entities are available for triggering like a one-time charge of the water heater.
+
+### Number
+
+Number entities are available to adjust values like the predefined temperature for different heating programs or the heating curve shift and slope.
+
+## Services
+
+The following services of the [climate integration](/integrations/climate/) are provided by the ViCare integration: `set_temperature`, `set_hvac_mode`, `set_preset_mode` 
 
-## Climate
+The following services of the [water_heater integration](/integrations/water_heater/) are provided by the ViCare integration: `set_temperature`
 
-A note about the current temperature attribute: Viessmann devices with room temperature sensing will show the current room temperature via `current_temperature`. All other devices will show the current supply temperature of the heating circuit.
+### Service `vicare.set_vicare_mode`
 
-### Supported services `climate.vicare_heating`
+Set the mode for the climate device as defined by Viessmann (see [set_hvac_mode](#service-climateset_hvac_mode) for a mapping to Home Assistant Climate modes. This allows more-fine grained control of the heating modes.
 
-The following services of the [Climate integration](/integrations/climate/) are provided by the ViCare {% term integration %}: `set_temperature`, `set_hvac_mode`, `set_preset_mode` 
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id` | yes | String or list of strings that point at `entity_id`'s of climate devices to control. To target all entities, use `all` keyword instead of entity_id.
```

Comment:
```suggestion
| `entity_id` | yes | String or list of strings that point at `entity_id`'s of climate devices to control. To target all entities, use the `all` keyword instead of entity_id.
```